test/unit/spec/PollController_spec.js

```diff
@@ -44,21 +42,23 @@ function (Okta, Q, OktaAuth, Util, PollingForm, Expect, Router, $sandbox, resPol
   }
 
   Expect.describe('Polling', function () {
-    describe('PollingForm Content', function () {
-      itp('shows the correct content on load', function () {
+    describe('Form Content', function () {
+      it('shows the correct content on load', function (done) {
         return setup().then(function (test) {
-          const title = 'There are too many users trying to sign in right now. We will automatically retry in 2 seconds.';
+          const title = 'There are too many users trying to sign in right now. We will automatically retry in 1 seconds.';
           expect(test.form.pageTitle().text().trim()).toBe(title);
-        });
+          done();
```

Comment:
Why are we using done here ? There is nothing async here right.

Reply:
these tests were continuing to poll and make network requests after it ended. I wanted some way for these to stop polling after the test is over.

Reply:
https://jasmine.github.io/tutorials/a